LEAF_Request_Portal/templates/reports/LEAF_import_data.tpl

```diff
@@ -451,9 +456,11 @@
                 .dialog( "close" );
             progressbar.progressbar( "value", false );
             progressLabel.text( "Starting import..." );
+            progressbar.progressbar( "value", 0);
         }
 
         function progress() {
+            debugger;

```

Comment:
Debugger should be turned off for production.

Reply:
Also not sure if we want the console output to still be on.
![Screen Shot 2020-10-05 at 12 52 42 PM](https://user-images.githubusercontent.com/2892376/95108759-b0f6da80-0709-11eb-943b-812b32785a43.png)